src/view/components/HtmlView/HtmlView.tsx

```diff
@@ -4,14 +4,14 @@ import {classnames} from '@bem-react/classnames';
 
 import {ClassNameProps} from '../../../classname';
 
-export type YfmHtmlProps = ClassNameProps & {
+export type HtmlViewProps = ClassNameProps & {
     html: string;
     linksVisited?: boolean;
     noListReset?: boolean;
     qa?: string;
 };
 
-export const YfmHtml = forwardRef<HTMLDivElement, YfmHtmlProps>(function YfmHtml(props, ref) {
+export const HtmlView = forwardRef<HTMLDivElement, HtmlViewProps>(function HtmlView(props, ref) {
```

Comment:
Let's rename it to `YFMView` or `YFMStaticView` and export it with an additional old name for backward compatibility

Reply:
fixed